tv_apps/smart-home/js/home.js

```diff
@@ -176,6 +176,22 @@
       });
     },
 
+    _localizeCardName: function(elem, card) {
+      if (!elem || !card) {
+        return;
+      }
+
+      // We should use user given name first, otherwise we use localized
+      // application/deck name.
+      var lang = document.documentElement.lang;
+      var name = this.cardManager.resolveCardName(card, lang);
+      if (name && name.raw) {
+        elem.textContent = name.raw;
```

Comment:
you may want to also remove attribute data-l10n-id if this can happen on an element that previously had data-l10n-id.